jinjasql/core.py

```diff
@@ -181,7 +181,6 @@ def __init__(self, env=None, param_style='format', identifier_quote_character='"
     def _prepare_environment(self):
         self.env.autoescape=True
         self.env.add_extension(SqlExtension)
-        self.env.add_extension('jinja2.ext.autoescape')
```

Comment:
@jpy-git 

According to the jinja2 changelog this feature was deprecated in 3.0.0 and removed in 3.1.0.
- https://github.com/pallets/jinja/blob/main/CHANGES.rst#version-300

But the proposed requirements here allow Jinja2>=2.8.

We can support this range by wrapping the `add_extension` call in a `try/except`

```python
try:
    self.env.add_extension('jinja2.ext.autoescape')
except AttributeError:
    pass
```

Reply:
It was only deprecated in 3.0.0, it hasn't been required since 2.9 so I should have made it `Jinja2>=2.9`
https://github.com/pallets/jinja/commit/a27a503cea8779db67a1bc21218a2e05e20ce6fc

Reply:
Happy to have the `try/except` block for people using `Jinja<2.9` though